### PR TITLE
updated readme to include current state section and be a little more …

### PR DESCRIPTION
**Pull request by Deadairx**
…understandable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ How this is done:
 3. Invoke `tsqlust::get_diagnostics_for_tsql(&your_sql)`
 4. Iterate over the resulting `Vec<tsqlust::diagnostics::Diagnostic>` and take action
 
+## Current State of Project
+
+> Currently the project only lexes and parses a query up to a WHERE statement
```
Comment (phrohdoh) [inline README.md]: `up to a WHERE statement` isn't quite true (it implies there is a linear flow of sql statements).

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 ## Project Goals
 
-> Provide developers with the ability to statically verify and analyze T-SQL queries
+> Provide developers human-friendly messages, errors and warnings when writing sql queries.
```
Comment (phrohdoh) [inline README.md]: Please say `tsql` instead of `sql`.

**Comment by Deadairx** [comment]
Updated per your notes @Phrohdoh 
Let me know if you have any other tweaks

**Review by phrohdoh — APPROVED** [review]
LGTM, thanks!

**Pull request merged**